x-pack/plugins/security_solution/cypress/e2e/cases/creation.cy.ts

```diff
@@ -120,5 +122,12 @@ describe('Cases', () => {
     cy.get(TIMELINE_TITLE).contains(this.mycase.timeline.title);
     cy.get(TIMELINE_DESCRIPTION).contains(this.mycase.timeline.description);
     cy.get(TIMELINE_QUERY).should('have.text', this.mycase.timeline.query);
+
+    cy.visit(OVERVIEW_URL);
+    cy.get(OVERVIEW_CASE_NAME).should('have.text', this.mycase.name);
+    cy.get(OVERVIEW_CASE_DESCRIPTION).should(
+      'have.text',
+      `${this.mycase.description} ${this.mycase.timeline.title}`
+    );
```

Comment:
```suggestion
    it('Should appear in overview page', () => {
        cy.visit(OVERVIEW_URL);
        cy.get(OVERVIEW_CASE_NAME).should('have.text', this.mycase.name);
        cy.get(OVERVIEW_CASE_DESCRIPTION).should(
          'have.text',
          `${this.mycase.description} ${this.mycase.timeline.title}`
        );
    });
```

Reply:
Thanks for the suggestion @semd but we cannot do it like that because tests should be independent between them. If we execute.  it('Should appear in overview page') in isolation, is going to fail because the case has not been created yet. It would also fail in case there is a problem with the previous test.

Reply:
Notice the `it('Should appear in overview page'` that I am suggesting is nested inside the `it('Creates a new case with timeline and opens the timeline'` main test, the Case will exist there. 
I was only considering adding some context because the main test refers only to the Timeline, and we are adding a non-related check for the Case on the Overview page, if this part fails the test name may be misleading and cause confusion among engineers.
We could create new a long isolated test, but we can also leverage the existing one, as you did, I just suggested wrapping it with a descriptive `it` title, if it fails both titles will be displayed.
But never mind, it is just an NIT

Reply:
@MadameSheema I've tested my suggestion in your branch, and it needs more work than what I pointed out, we'd have to extract the Case creation logic in a previous `before` block and then separate the test cases into two different `it`, being aware of the `this.mycase` scope. I thought it would be simpler. 
Feel free to ignore my comment.
Thanks for doing the test, let's get this merged! 🚀